visualization/view.py

```diff
@@ -0,0 +1,153 @@
+            print(e)
+    else:
+        #try:
+            bench_names = ["Benchmark #1", "Benchmark #2"]
+            bench_fid = {"Benchmark #1" : 0 , "Benchmark #2" : 0}
+            count1 = 0
+            count2 = 0
+            avgs = []
+            for i, bench in enumerate(contents["benchmarks"]):
+                count1+=1
+                bench_fid["Benchmark #1"] = bench_fid.get("Benchmark #1") + bench["stats"]["quality"]["fidelity"]
+                
+            for i, bench in enumerate(contents2["benchmarks"]):
+                count2+=1
+                bench_fid["Benchmark #2"] = bench_fid.get("Benchmark #2") + bench["stats"]["quality"]["fidelity"]
+            
+            avgs.append(bench_fid["Benchmark #1"]/count1)
+            avgs.append(bench_fid["Benchmark #2"]/count2)
+            print(avgs)
+            return bench_names, avgs
+        #except Exception as e:
+            #print("Uh, Oh! Something went wrong")
+            #print(e)
+                
+def meantime(contents, contents2 = None):
+    if not contents2:
+        try:
+            bench_name = {}
+            bench_mean= {}
+            for i,bench in enumerate(contents["benchmarks"]):
+                bench_name[bench["name"]] = i + 1
+                if bench_mean.get(bench["name"]):
+                    bench_mean[bench["name"]] = bench_mean.get(bench["name"]) + bench["stats"]["timing"]["mean"]
+                else:
+                    bench_mean[bench["name"]] = bench["stats"]["timing"]["mean"]
+            timeavg = []
+            for i in bench_mean:
+                timeavg.append(bench_mean[i]/bench_name[i])
+            return bench_name.keys(), timeavg
+        except Exception as e:
+            print("Uh, Oh! Something went wrong")
+            print(e)
+    else:
+        try:
+            bench_names = ["Benchmark #1", "Benchmark #2"]
+            bench_times = {"Benchmark #1" : 0, "Benchmark #2" : 0}
+            count1 = 0
+            count2 = 0
+            avgs = []
+            for bench in enumerate(contents["benchmarks"]):
+                count1+=1
+                bench_times["Benchmark #1"] += bench["stats"]["timing"]["mean"]
+                
+            for bench in enumerate(contents2["benchmarks"]):
+                count2+=1
+                bench_times["Benchmark #1"] += bench["stats"]["timing"]["mean"]
+            
+            avgs.append(bench_times["Benchmark #1"]/count1)
+            avgs.append(bench_times["Benchmark #2"]/count2)
+            return bench_names, avgs
+        except Exception as e:
+            print("Uh, Oh! Something went wrong")
+            print(e)
+            
+            
+        
+
+def main(args):
+    #arg parsing
+    if len(args) == 4:
+        viewtype = args[1]
+        file = args[2]
+        datatype = args[3]
+    if len(args) == 5:
+        viewtype = args[1]
+        file = args[2]
+        file2 = args[3]
+        datatype = args[4]
```

Comment:
Can you leverage argparse for this so we have a `--help` to explain the options to users?

Reply:
If need be, I could help you with this @danielleodigie.